backend/cfg/dataflow.mli

```diff
@@ -0,0 +1,42 @@
+module type Domain = sig
```

Comment:
Would this clash with Domain in multicore code?  The one recently added upstream uses all capitals for the name. Alternative name would be Abstract_domain or Lattice.

Reply:
That would shadow it in the current module (which is presumably
fine), and from the outside it will be `Dataflow.Domain`.